infra/token-transfer-client/src/components/pages/Dashboard.js

```diff
@@ -28,16 +28,11 @@ const Dashboard = props => {
   const displayLockupCta =
     nextVest && data.config.earlyLockupsEnabled && !data.config.isLocked
   const displayFullWidthLockupCta = displayLockupCta && hasLockups
```

Comment:
This or the above condition will need to be changed otherwise the CTA will never show. I think one or the other should just be set to `true` depending on if we want the full width CTA or the square box one.

Reply:
My understanding was that the square one would be displayed for people who had no existing lock-ups and the full-width banner would display if the square box was displaying existing lock-up information.

Reply:
Ok then I think just the `data.config.earlyLockupsEnabled` needs to be removed from that conditional.